src/perfwrapper/perfwrapper.c

```diff
@@ -103,89 +96,111 @@ int main(int argc, char **argv)
 			exit(EXIT_SUCCESS);
 		case '?':
 		default:
-			error("Wrong option argument\n");
-			error("%s", usage);
+			nrm_log_error("Wrong option argument\n");
+			nrm_log_error("%s", usage);
```

Comment:
can't use log here, before nrm_init. You can switch to fprintf(stderr) or put nrm_init/nrm_log_init higher in the function.